src/Microsoft.Health.Dicom.Core/Registration/DicomFunctionsBuilderCoreRegistrationExtensions.cs

```diff
@@ -0,0 +1,30 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using EnsureThat;
+using Microsoft.Health.Dicom.Core.Extensions;
+using Microsoft.Health.Dicom.Core.Features.Common;
+using Microsoft.Health.Dicom.Core.Features.ExtendedQueryTag;
+using Microsoft.Health.Dicom.Core.Features.Indexing;
+
+namespace Microsoft.Health.Dicom.Core.Registration
+{
+    public static class DicomFunctionsBuilderCoreRegistrationExtensions
```

Comment:
We need naming guidlines :-)
We don't predix DICOM any where in this solution. Because everything is dicom.

I don't understand this class name. Is this to consolidate all ExtendedQueryTag related services? We should name accordingly

Reply:
It imitates [`DicomServerBuilderSqlServerRegistrationExtensions`](https://github.com/microsoft/dicom-server/blob/features/ExtendedQueryTagsV2/src/Microsoft.Health.Dicom.Functions/Indexing/ReindexDurableFunction.Trigger.cs#L65), the difference is this one is for services/components in `Core`, while `DicomServerBuilderSqlServerRegistrationExtensions` is for `SqlServer`. 

I feel we probably should refactor how we build DicomServer/DicomFunctions, but I don't have a good idea now, will think about that.

Reply:
The class has been removed.